try {
    adggag
}

catch(error){
    console.log("Hubo un error")
}; 